.gitignore

```diff
@@ -109,3 +109,5 @@ version.texi
 
 no_llvm_build
 
+**node_modules/
+**doc-ui/.lock
```

Comment:
Hm, do these need to be this much of a blanket ignore? Perhaps we can isolate it to some subpath?

(Very much not important if you don't feel like dealing with it).